pkg/serializer/internal/metrics/sketch_series_test.go

```diff
@@ -155,8 +155,8 @@ func TestSketchSeriesMarshalSplitCompress(t *testing.T) {
 		sl.Append(Makeseries(i))
 	}
 
-	serializer1 := SketchSeriesList{SketchesSource: sl}
-	payload, _ := serializer1.Marshal()
+	// serializer1 := SketchSeriesList{SketchesSource: sl}
```

Comment:
My vote is to just remove these. Commented code is not very desirable IMO, if we are ever curious about this we can check the git history.